Automattic/newspack-plugin
# Add Material UI package to the dependencies and update all SVG to use icons from `@material-ui/icons`

### All Submissions:






### Changes proposed in this Pull Request:

Since the icons used around the plugin are coming from Material, I thought it would make sense to add `material-ui` as a dependency.

That way we can simply import the icon that we need instead of relying on SVG and Path.

I've removed all the different icons that were being displayed via `import { SVG, Path } from '@wordpress/components';` and replaced them with `import *Icon from '@material-ui/icons/*';`

### How to test the changes in this Pull Request:

1. Switch to this branch
2. Check all the pages
3. Do you see the icons everywhere (Title, checkboxes, notices etc...)?

### Other information:
